e2e/nctl/RPC.test.ts

```diff
@@ -88,8 +89,11 @@ describe('RPC', () => {
   });
 
   xit('queryBalance', async () => {
-    const res = await client.queryBalance(PurseIdentifier.PurseUref, 'uref-6f2d3316f5c114923e6ec7087a399f692f4ce85197106a6daa97bf7f444e4f9e-007');
-    assert.equal(res.toString(), "1000000000000000000000000000000000");
+    const res = await client.queryBalance(
+      PurseIdentifier.PurseUref,
+      'uref-6f2d3316f5c114923e6ec7087a399f692f4ce85197106a6daa97bf7f444e4f9e-007'
+    );
+    assert.equal(res.toString(), '1000000000000000000000000000000000');
```

Comment:
I realize you didn't write this, but could we dynamically generate this string? i.e. `assert.equal(res.toString(), `1${'0'.repeat(33)}`);`. It's a bit cleaner and then we have easy control over the size.

Reply:
Agreed with you this point, but I think the refactoring isn't the purpose of this PR as you can see.
For refactoring test cases, I think it can be done in the other PR.
We shouldn't make unnecessary changes in the PR since it makes reviewing more difficult.
I will try to refactor this in other PR, anyway thanks for mentioning this 👍